docs/GettingStarted/configuration/PelorusCore.md

```diff
@@ -95,7 +95,7 @@ For detailed information about planning Prometheus storage capacity and configur
 
 Unlike ephemeral volume that have a lifetime of a pod, persistent volume allows to withstand container restarts or crashes making Prometheus data resilient to such situations. Pelorus allows to use underlying [Prometheus Operator Storage](https://prometheus-operator.dev/docs/operator/storage/#storage-provisioning-on-aws) capabilities by using Kubernetes [`StorageClass`](https://kubernetes.io/docs/concepts/storage/storage-classes/).
 
-It is recommended to use Prometheus Persistent Volume **together** with the [Thanos](#thanos) for the long term storage.
+It is recommended to use Prometheus Persistent Volume **together** with [Thanos](#thanos) for the long term storage. Check [NooBaa for Long Term Storage](Noobaa.md) for more information.
```

Comment:
I would NOT put the `Check [NooBaa for Long Term Storage](Noobaa.md) for more information.` in this section. NooBaa is an example of S3 provider and users may choose other S3 provider/bucket, so we do not *recommend* using NooBaa here. What we *do* recommend is to use Thanos. Under Thanos section we do have sentence now:
`If you don't have an object storage provider, we recommend NooBaa as a free, open source option. You can check [NooBaa for Long Term Storage](Noobaa.md) to guide on how to host an instance on OpenShift and configure Pelorus to use it.`

Reply:
alright, I will remove from here